web/src/app/services/data-store/data-store.service.ts

```diff
@@ -312,34 +313,68 @@ export class DataStoreService {
   }
 
   lois$({id}: Survey, {email}: User): Observable<List<LocationOfInterest>> {
-    const predefinedLois = this.db.collection(
-      `${SURVEYS_COLLECTION_NAME}/${id}/lois`,
-      ref => ref.where('predefined', '==', true)
-    );
-
-    const userLois = this.db.collection(
-      `${SURVEYS_COLLECTION_NAME}/${id}/lois`,
-      ref =>
+    return this.db
+      .collection(SURVEYS_COLLECTION_NAME, ref =>
         ref
-          .where('predefined', '==', false)
-          .where('created.user.email', '==', email)
-    );
-
-    return combineLatest([
-      predefinedLois.valueChanges({idField: 'id'}),
-      userLois.valueChanges({idField: 'id'}),
-    ]).pipe(
-      map(lois => {
-        const [predefinedLois, userLois] = lois;
-        const combined = predefinedLois.concat(userLois);
-        return List(
-          combined
-            .map(obj => LoiDataConverter.toLocationOfInterest(obj.id, obj))
-            .filter(DataStoreService.filterAndLogError<LocationOfInterest>)
-            .map(loi => loi as LocationOfInterest)
-        );
-      })
-    );
+          .where(documentId(), '==', id)
+          .where(new FieldPath('acl', email), '==', Role.SURVEY_ORGANIZER)
```

Comment:
We have the survey in the method args, can we get the acl from there instead of loading it again? Ideally we would pass the user's role into the method.